src/components/Grid.jsx

```diff
@@ -33,6 +33,22 @@ class Grid extends React.Component {
     }
   }
 
+  getLines () {
+    let lines = []
+
+    for (let i = 0; i < this.props.hours; i++) {
```

Comment:
Why map here. Isn't this more understandable when i'm working with sequence of numbers?